test/verify-sparse-labels.py

```diff
@@ -0,0 +1,44 @@
+"""Validate that the output produced by integration testing on 'label-maker labels' matches our expectations"""
+import json
+import numpy as np
+
+# our labels should look like this
+expected_labels = {
+    '62092-50162-17': np.array([1, 0, 0, 0, 0, 0, 0]),
```

Comment:
We should actually expect not to see this first one, correct?

Reply:
Deleting it didn't cause checks to fail. That was weird, but it was because we just check that every tile in the npz file matches something in expected labels. I added a line (here and in vanilla `verify-labels.py` to make sure that the number of tiles in `label.npz` and `expected_labels` also match